src/main/java/org/jitsi/jicofo/FocusManager.java

```diff
@@ -117,13 +118,13 @@
     /**
      * The thread that expires {@link JitsiMeetConference}s.
      */
-    private FocusExpireThread expireThread = new FocusExpireThread();
+    private final FocusExpireThread expireThread = new FocusExpireThread();
 
     /**
      * Jitsi Meet conferences mapped by MUC room names.
      */
-    private Map<String, JitsiMeetConference> conferences
-        = new HashMap<String, JitsiMeetConference>();
+    private final Map<String, JitsiMeetConference> conferences
+        = new ConcurrentHashMap<>();
```

Comment:
Is that because of getConferenceCount()? There are explicit `synchronized (this)` (including a `synchronized (FocusManager.this)`) which seem to be introduced for the sole purpose of protecting `conferences` so adding ConcurrentHashMap results in an unnecessary confusion or at the very least a doubt.